subprojects/docs/src/docs/userguide/jvm/test_report_aggregation_plugin.adoc

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[test_report_aggregation_plugin]]
+= The Test Report Aggregation Plugin
+
+The Test Report Aggregation plugin (plugin id: `test-report-aggregation`) provides tasks and configurations which can be used to aggregate the results of multiple link:{groovyDslPath}/org.gradle.api.tasks.testing.Test.html[Test] task invocations into a single HTML report.  This aggregation is done in a way that is compatible with the <<configuration_cache#configuration_cache,configuration cache>>.
```

Comment:
I'd even drop the "tasks and configurations" bit from the first sentence - just describe the purpose of the plugin, not say anything about the mechanism to begin with.